Neo4j.Driver/Neo4j.Driver.IntegrationTests/DirectDriver/SessionIT.cs

```diff
@@ -46,6 +47,20 @@ public void DisallowNewSessionAfterDriverDispose()
         }
 
         [Fact]
+        public void DisallowRunInSessionAfterDriverDispose()
+        {
+            var driver = GraphDatabase.Driver(ServerEndPoint, AuthToken);
+            var session = driver.Session(AccessMode.Write);
+            session.Run("RETURN 1").Single()[0].ValueAs<int>().Should().Be(1);
+
+            driver.Dispose();
+
+            var error = Record.Exception(() => session.Run("RETURN 1"));
+            error.Should().BeOfType<ObjectDisposedException>();
```

Comment:
Is it possible to throw some sort of `DriverDisposedException` in this case? Is it the only way connection pool can be disposed?